main.go

```diff
@@ -13,6 +14,10 @@ import (
 	"google.golang.org/api/option"
 )
 
+const (
+	credentialFilePath = "credentials"
```

Comment:
@bradrydzewski We need to store credentials to a file. Which file path can be used that can work for all the os architectures?

Reply:
hey Shubham, I would recommend using the [ioutil.TempDir](https://golang.org/pkg/io/ioutil/#TempDir) function to create a temporary directory (this will work on all operating systems) and then create the credentials file in this directory.

Reply:
Done

Reply:
Working with os.MkdirAll temp directory creation